src/collective/collectionfilter/portlets/__init__.py

```diff
@@ -27,4 +30,9 @@ def reload_url(self):
 
     @property
     def available(self):
+        if not (
```

Comment:
Please add a comment why you are disabling the availability for those interfaces here.

Reply:
@thet 

Thanks for comment!

Use Case:

1. enable Collection Behavior for Folders
2. add a Folder
3. add some Documents to the Folder with different Subjects
4. configure the Collection, e.g. show only Documents
5. assign a Collection Filter Portlet to the Folder, use "Subject" as Criteria, point to the Folder itself
6. Filter the Collection via Portlets and Click on a Result Item

Now the Portlets are empty or it make no sense, that the Filter Options available on the Children Detail View. That is the reason why i restricted the rendering to the Interfaces "ISyndicatableCollection" and "ICollectionish". It means, only if the Context a Collection, show the Portlets. Maybe this to strictly ?